### ISSUE TITLE
Help Apprentice Harper key android

### ISSUE THREAD

**Issue by BRASPO22** [noDRM/DeDRM_tools#354]
### Question / bug report

Harper's Apprentice

Is there any alternative for android key backup other than by adb?
That alternative no longer works for me.
backup.aba only 1kb in size.
Please have another solution?
I've tried it on different android versions and the older Kindle version indicated by you and it doesn't go beyond that value.
thanks in advance.

### Which version of Calibre are you running?

6.14

### Which version of the DeDRM plugin are you running?

V7

### Log output

```log
Paste log output here.
```

**Comment by BRASPO22** [comment]
Hi thank you so much for replying.
I used Kindle android 4.16.0.75 it worked perfectly for a week and during that time the app was updated 3 times I reinstalled the old version and I generated 3 keys that worked perfectly.
I confess that I abused a lot and converted many ebooks (about 300 at least).
Amazon I think it got it.
After several attempts, I downloaded 4.15.0.46 and it still doesn't generate the key.
When I make the backup the cell phone says that it was done, the file does not appear, but if I restart the computer the file appears with 1Kb.

**Comment by ElleKayEm** [comment]
Yeah, a 1kb backup file is not a real backup file.  All I can suggest is looking over the post I linked earlier to see what others have had success with.

**Comment by BRASPO22** [comment]
I do the whole process on my cell phone because my cell phone is old (Android 6.0.1) I tried it on an older cell phone too (android 4.1.2) and it also didn't work. I just followed the steps that Apprentice Harper posted here and it worked for me. The more elaborate steps in the link you posted I didn't do. My problem is the key that does not exceed 1Kb, since the file size was larger before. Thank you very much

**Comment by BRASPO22** [comment]
Thank you very much for your attention.
you are amazing.

**Comment by ElleKayEm** [comment]
Have you got automatic app updates shut off?

**Comment by BRASPO22** [comment]
No.

**Comment by ElleKayEm** [comment]
May be updating to a version that doesn't allow backups.  People seem to have better luck with bluestacks.

**Comment by BRASPO22** [comment]
To close the topic, as you guys are always sharing your discoveries with us, I wanted to share something I noticed using my old android phone.
But what I'm going to report is more information for you, in case it's useful in any way.

Can I report and it won't be posted here?

**Comment by ElleKayEm** [comment]
You could email noDRM (the owner of this repo).  Email is in their profile here.  But why not post here?  Could be useful for anyone interested in contributing to this project.
